commands:
  - linear_x:  0.0
    linear_y:  0.0
    angular_z: 0.0
    duration: 5.0

  - linear_x:  0.6
    linear_y:  0.0
    angular_z: 0.05
    duration: 200.0
